README.md

```diff
@@ -29,20 +29,16 @@ Refract is a recursive data structure for expressing complex structures, relatio
 
 Refract is a structure for handling all sorts and kinds of data across all sorts and kinds of formats. That's a very general-purpose description for a general-purpose structure. To get an idea of what Refract does, we'll walk through some of its uses with some examples along the way.
 
-### As a way to annotate JSON
+### As a way to annotate data
 
-Refract provides the ability to take a normal JSON structure and add a layer on top of it for the puprose of annotating and adding semantic data. Refract is not the first structure to try and tackle this task, though it does take a different approach. Instead of creating an entirely different structure to describe the data, Refract's approach is to expand the existing structure (we call it "refracting" a structure). Here is an example to show our point.
+Refract provides the ability to take data structures and add a layer on top of it for the purpose of annotating and adding semantic data. Refract is not the first structure to try and tackle this task, though it does take a different approach. Instead of creating an entirely different structure to describe the data, Refract's approach is to expand the existing structure (we call it "refracting" a structure). Here is an example to show our point.
 
-Take the following simple JSON object.
+Take the following simple object which contains a name and email.
 
-```json
-{
-  "name": "John Doe",
-  "email": "john@example.com"
-}
-```
+- name: John Doe
+- email: john@example.com
```

Comment:
Yeah, I think you're right. You could make the distinction that it's a JSON example.